ic-agent/src/identity/mod.rs

```diff
@@ -7,6 +7,7 @@ pub use basic::{BasicIdentity, PemError};
 
 #[derive(Clone, Debug)]
 pub struct Signature {
+    pub der_encoded_public_key: Vec<u8>,
```

Comment:
This would have to be added to all identity signers. Is this required from the spec now?

Reply:
We could leave the field `public_key`, but it would have to be understood that they are DER-encoded.

Reply:
Removed this field, and noted that `public_key` must be DER-encoded.